internal/batches/sources/sources.go

```diff
@@ -541,7 +570,7 @@ func getCloneURL(repo *types.Repo) (*vcs.URL, error) {
 		parsedURLs = append(parsedURLs, parsedURL)
 	}
 
-	sort.SliceStable(parsedURLs, func(i, j int) bool {
+	sort.SliceStable(parsedURLs, func(i, _ int) bool {
 		return !parsedURLs[i].IsSSH()
 	})
```

Comment:
unrelated to this change, but I don't think this sort is guaranteed to work as intended. The thunk passed to `SliceStable` is a "less" function, so, for example, if neither `parsedURLs[i]` and `parsedURLs[j]` are SSH URLs, `i` will be sorted before `j` even though they are actually "equal" by this comparison.

```suggestion
	sort.SliceStable(parsedURLs, func(i, _ int) bool {
		return !parsedURLs[i].IsSSH() && parsedURLs[j].IsSSH()
	})
```

Reply:
Oh, nice catch.  I didn't know that.
Thanks for flagging.